linux57-tkg/install.sh

```diff
@@ -132,74 +167,118 @@ if [ "$1" = "install" ]; then
+      mv ~/rpmbuild/RPMS/x86_64/* "$_where"/RPMS/
+
+      #Clean up the original folder, unneeded and takes a lot of space
+      rm -rf ~/rpmbuild/
+
+      read -p "Do you want to install the new Kernel ? y/[n]: " _install
+      if [ "$_install" = "y" ] || [ "$_install" = "Y" ] || [ "$_install" = "yes" ] || [ "$_install" = "Yes" ]; then
+        
+        _kernelname=$_basekernel.${_kernel_subver}_$_kernel_flavor
+        _headers_rpm="kernel-headers-${_kernelname}*.rpm"
+        _kernel_rpm="kernel-${_kernelname}*.rpm"
+        _kernel_devel_rpm="kernel-devel-${_kernelname}*.rpm"
+        
+        if [ "$_distro" = "Fedora" ]; then
+          sudo dnf install RPMS/$_headers_rpm RPMS/$_kernel_rpm RPMS/$_kernel_devel_rpm
+        elif [ "$_distro" = "Suse" ]; then
+          msg2 "Some files from 'linux-glibc-devel' will be replaced by files from the custom kernel-hearders package"
+          msg2 "To revert back to the original kernel headers do 'sudo zypper install -f linux-glibc-devel'" 
+          sudo zypper install --replacefiles --allow-unsigned-rpm RPMS/$_headers_rpm RPMS/$_kernel_rpm RPMS/$_kernel_devel_rpm
+        fi
+        
+        msg2 "Install successful" 
       fi
     fi
   fi
+
 fi
 
 if [ "$1" = "uninstall" ]; then
 
   cd "$_where"
+  msg2 "List of installed custom tkg kernels: "
 
-  if [ ! -f installed-kernels ] || [ ! -s installed-kernels ]; then
-    echo "No custom kernel has been installed yet"
-    exit 0
-  fi
-
-  i=1
-  declare -a _custom_kernels
-  msg2 "Installed custom kernel versions: "
-  while read p; do
-    echo "    $i) $p"
-    _custom_kernels+=($p)
-    i=$((i+1))
-  done < installed-kernels
-  
-  i=$((i-1))
-  _delete_index=0
-  read -p "Which one would you like to delete ? [1-$i]: " _delete_index
-
-  if [ $_delete_index -ge 1 ] && [ $_delete_index -le $i ]; then
-    _delete_index=$((_delete_index-1))
-    sudo dpkg -r linux-headers-${_custom_kernels[$_delete_index]} linux-image-${_custom_kernels[$_delete_index]}
+  if [ "$_distro" = "Ubuntu" ]; then
+    dpkg -l "*tkg*" | grep "linux.*tkg"
+    dpkg -l "*linux-libc-dev*" | grep "linux.*tkg"
+    msg2 "To uninstall a version, you should remove the linux-image, linux-headers and linux-libc-dev associated to it (if installed), with: "
+    msg2 "      sudo apt remove linux-image-VERSION linux-headers-VERSION linux-libc-dev-VERSION"
+    msg2 "       where VERSION is displayed in the lists above, uninstall only versions that have \"tkg\" in its name"
+  elif [ "$_distro" = "Fedora" ]; then
+    dnf list --installed kernel*
+    msg2 "To uninstall a version, you should remove the kernel, kernel-headers and kernel-devel associated to it (if installed), with: "
+    msg2 "      sudo dnf remove --noautoremove kernel-VERSION kernel-devel-VERSION kernel-headers-VERSION"
+    msg2 "       where VERSION is displayed in the second column"
+  elif [ "$_distro" = "Suse" ]; then
+    zypper packages --installed-only | grep kernel.*tkg
```

Comment:
`kernel.*tkg` should be quoted for consistency with the other `grep`s.

Reply:
done!